src/components/TranslationChartImage.tsx

```diff
@@ -1,39 +1,19 @@
-import React from "react"
 import { useColorModeValue } from "@chakra-ui/react"
-import { useStaticQuery, graphql } from "gatsby"
-import GatsbyImage from "./GatsbyImage"
-import { getImage } from "../utils/image"
+import { Image } from "@/components/Image"
+import pageviewsLight from "@/public/translation-program/pageviews-light.png"
+import pageviewsDark from "@/public/translation-program/pageviews-dark.png"
```

Comment:
```suggestion
import pageviewsDark from "./translation-program/pageviews-dark.png"
```

When importing stuff from `public` dir, relative path is enough